packages/demo/src/PostAddNext.re

```diff
@@ -23,6 +23,27 @@ module PostAddMutationConfig = [%graphql
 module PostAddMutation =
   ReasonApolloHooks.Mutation.Make(PostAddMutationConfig);
 
+module FieldString = {
+  [@react.component]
+  let make = (~field, ~label) => {
+    <PostAddForm.Field
+      field
+      render={({handleChange, error, value, validate}) => {
+        Js.log(label);
+        <label>
+          <span> {React.string(label)} </span>
+          <input
+            value
+            onChange={Helpers.handleChange(handleChange)}
```

Comment:
what about create a module DomHelpers? 🤔 just because of the naming, this function is needed only for web right?

Reply:
indeed this one is way to generic